_config.yml

```diff
@@ -4,12 +4,20 @@ tagline: The compiler for writing next generation JavaScript
 description: The compiler for writing next generation JavaScript
 baseurl: "" # the subpath of your site, e.g. /blog/
 url: "https://babeljs.io" # the base hostname & protocol for your site
-# twitter_username: jekyllrb
+twitter_username: babeljs
```

Comment:
It's fine to set this, but it doesn't really do much atm since we don't actually use these social vars anywhere.